### ISSUE TITLE
Editing existing monitor does not work

### ISSUE THREAD

**Issue by brennerm** [jpavlav/UptimeRobotPy#13]
Hey @jpavlav, thanks for UptimeRobotPy. It has been of great use for me. Unfortunately editing existing monitors does not seem to work or I'm doing something wrong. Steps to reproduce:

1. create a monitor (`new_monitor`) with an interval of e.g. 600
2. update this monitor (`up_robot.edit_monitor(identifier, interval=1200)`) with an interval of e.g. 1200
3. query the monitor (`get_all_monitors`)
4. the interval of the monitor is still at 600

This seems to apply to all parameters except `friendly_name` which works correctly.

Tried to change the interval using a cURL request which worked well so it's probably not an upstream issue. Let me know if you need more info.

**Comment by jpavlav** [comment]
Hey @brennerm,

Glad it's been useful for you! I'll see if I can suss out what's going on and get back to you.

**Comment by jpavlav** [comment]
Hello again @brennerm,

So, I believe I figured out the issue:

```
In [2]: from uptimerobotpy import UptimeRobot

In [3]: up_robot = UptimeRobot(api_key=UPTIME_ROBOT_API_KEY)

In [4]: monitor = up_robot.get_monitors()['monitors'][0]

In [5]: monitor.get('interval')
Out[5]: 1200

In [6]: edit_resp1 = up_robot.edit_monitor(monitor['id'], interval=600)

In [7]: monitor = up_robot.get_monitors()['monitors'][0]

In [8]: monitor.get('interval')
Out[8]: 1200

In [9]: interval = "600"

In [10]: type(interval)
Out[10]: str

In [11]: edit_resp2 = up_robot.edit_monitor(monitor['id'], interval=interval)

In [12]: monitor = up_robot.get_monitors()['monitors'][0]

In [13]: monitor.get('interval')
Out[13]: 600
```

Looks like it expects the value of that parameter to be a string. A little surprising to me, considering it is returned as an interval. Having said that, it's a little surprising to me that it's accepted at all as something other than a string, if a string is what is expected. In truth, looking at the documentation, it appears as though the API has it's wires crossed about what it wants a bit.

If you look at the examples, they seem to include what appear to be URL parameters as a payload to a POST request. This lib/client sends the data as a json payload but, perhaps, if all of the payloads were passed as parameters to requests, it would work regardless because, afaik, that would always be a string.

TLDR; looks like they expect a string as the value for the `interval` param. Could be that way for all of them 🤷 .

**Comment by brennerm** [comment]
Seems to work. It's weird but definitely not an issue of UptimeRobotPy. Maybe it'd help others to mention this somewhere on the README.

Thanks!
